packages/shared/src/components/modals/common.tsx

```diff
@@ -64,6 +71,7 @@ export const modals = {
   [LazyModal.ReadingHistory]: ReadingHistoryModal,
   [LazyModal.SquadPromotion]: SquadPromotionModal,
   [LazyModal.LegoReferralCampaign]: LegoReferralCampaignModal,
+  [LazyModal.CreateSharedPost]: CreateSharedPostModal,
```

Comment:
Can we also remove `PostToSquadModal`? 

The new modal should replace it right, functionality wise? If yes we need to make sure `SharePostChecklistStep` and `CompanionMenu` are updated to use this new modal.

Reply:
That's right. I was waiting for some answer from the design team on empty state that `PostToSquadModal` provides. I believe they have gotten back to it already. I will see its new implementation.